src/helpers/fixtures.js

```diff
@@ -60,7 +60,13 @@ let fixtureHelper = module.exports = {
         }
       }
 
-      if (options.query && !_.isEqual(options.query, fixture.request.query)) {
+      const filteredFixturedQuery = xhrHelper.getQueryStringObject({
+        url: 'http://localhost/?' + Object.keys(fixture.request.query || {}).map(function(key) {
+          return key + '=' + fixture.request.query[key];
+        }).join('&')
```

Comment:
If I use qs, fixtures don't match anymore. Possibly the `xhrHelper` is already escaping parameters, and using qs over this "dumb" approach ends up with them double-escaped?